src/main/java/ita/softserve/course_evaluation/service/FeedbackRequestService.java

```diff
@@ -19,7 +20,9 @@
 	Page<FeedbackRequestDto> findAllByCourseId(Pageable pageable, Long id);
 
 	List<FeedbackRequestDto> getFeedbackRequestByCourseIdOnly(long id);
-	
+
+	List<StudentFeedbackRequestDto> getFeedbackRequestByCourseIdAndStudentId(long idf, long ids);
+
```

Comment:
Use words that can be pronounced and avoid abbreviations.
It’s much clearer to say “studentId” than “ids”